failover-and-redundancy.md

```diff
@@ -1,24 +1,65 @@
 # Failover and redundancy
 
-This topic describes the failover characteristics and redundancy of Application Service Adapter components and applications that are pushed with Application Service Adapter. It includes the following sections:
+This topic describes the failover characteristics and redundancy of Application
+Service Adapter components and applications that are pushed with Application
+Service Adapter. It includes the following sections:
 
 * [Cloud Foundry-compatible API](#cloud-foundry-api)
 * [Controllers](#controllers)
 * [Applications](#applications)
 
+For instructions to modify the scaling characteristics of your Application
+Service Adapter installation, refer to the [scaling topic](scaling.md).
+
 ## <a id="cloud-foundry-api"></a>Cloud Foundry-compatible API
 
-Application Service Adapter deploys two replicas of a v3 Cloud Foundry-compatible API (Korifi API) that clients communicate with. This API is stateless and can be horizontally scaled for increased availability and performance.
+Application Service Adapter deploys two replicas of a v3 Cloud
+Foundry-compatible API (Korifi API) that clients communicate with. This API is
+stateless and can be [horizontally scaled](scaling.md#api) for increased
+availability and performance.
 
 ## <a id="controllers"></a>Controllers and Webhooks
 
-Application Service Adapter deploys multiple components known as [controllers](https://kubernetes.io/docs/concepts/architecture/controller/) to the cluster. These components watch and update state on the cluster in what is known as a "control loop" and over time ensure that the state in the cluster is consistent. Additionally, these components run [admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) that validate and update Application Service Adapter resources.
+Application Service Adapter deploys multiple components known as
+[controllers](https://kubernetes.io/docs/concepts/architecture/controller/) to
+the cluster. These components watch and update state on the cluster in what is
+known as a "control loop" and over time ensure that the state in the cluster is
+consistent. Additionally, these components run [admission
+webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
```

Comment:
Do you know if this will render ok with the link split across lines?

Reply:
Eh it looks fine on Github. I imagine it will look fine on the docs site.
https://github.com/pivotal/docs-tas-adapter/blob/a1a171168d412fed371a1f1b9dbd8a580d2eb5a5/failover-and-redundancy.md